src/l_7.py

```diff
@@ -0,0 +1,92 @@
+def duimvsant(duim):
+    sant = duim * 2.54
+    print("в" + duim + "дюймах " + sant + "сантиметров")
+    return sant, duim
```

Comment:
Пусть ф-я возвращает только результат

```suggestion
    return sant
```